plugins/MultiSites/vue/src/types.ts

```diff
@@ -27,11 +27,16 @@ export interface DashboardSiteData extends DashboardMetrics, Site {
   label: string;
 }
 
+export interface KPICardBadge {
+  label: string;
+  title: string;
```

Comment:
According to the tests I guess `title` is optional, right?
```suggestion
  title?: string;
```